### PR TITLE
 Eula Full screen

### PR DESCRIPTION
**Pull request by ArshdeepSingh3-eaton**
Fixes # .

#### Changes proposed in this Pull Request:

- Added Eula Full Screen
-
-

#### Screenshots / Screen Recording (if applicable)

-
Screenshot 2024-03-05 at 5 38 21 PM

#### To Test:

- in RegistrationProvider Example
````
<RegistrationWorkflow>
<EulaScreen/>
</RegistrationWorkflow>
````

#### Any specific feedback you are looking for?

-

### CODE REVIEW

File: login-workflow/docs/screens/eula.md [hunk 1]
```diff
@@ -23,6 +25,13 @@ import {  EulaScreenBase } from '@brightlayer-ui/react-native-auth-workflow';
 | initialCheckboxValue | `boolean` | Used to pre-populate the checked/unchecked checkbox when the screen loads. | `false` |
 | onEulaAcceptedChange | `(accepted: boolean) => void` | Called when the acceptance checkbox clicked. |  |
 | errorDisplayConfig | `ErrorManagerProps` | See [Error Management](../../components/error-manager.md) |  |
-| onRefetch | `() => void` | Function to refetch Eula content |  |
+| refreshConfig | `RefreshConfigProps` | configuration for refresh screen |  |
+
+This screen also extends the `WorkflowCardProps` type for updating the title, instructions, buttons, etc. See [Workflow Card](../../components/workflow-card.md) for more details.
```
[the remark this answers is not in the document]
Comment (surajeaton) [reply login-workflow/docs/screens/eula.md:30]: https://teams.microsoft.com/l/message/19:4cb6d8d0c70d4bd4a93a78b09a21c420@thread.skype/1709560192182?context=%7B%22contextType%22%3A%22chat%22%7D

File: login-workflow/src/__tests__/screens/EulaScreen-test.tsx [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+                        <RegistrationWorkflow initialScreenIndex={0}>
+                            <EulaScreen />
+                        </RegistrationWorkflow>
+                    </RegistrationContextProvider>
+                </SafeAreaProvider>
+            )
+            .toJSON();
+        expect(rendered).toBeTruthy();
+    });
+    it('on next press', () => {
```
[the remark this answers is not in the document]
Comment (surajeaton) [reply login-workflow/src/__tests__/screens/EulaScreen-test.tsx:29]: https://teams.microsoft.com/l/message/19:4cb6d8d0c70d4bd4a93a78b09a21c420@thread.skype/1709560192182?context=%7B%22contextType%22%3A%22chat%22%7D

**Comment by manojleaton** [comment]
Please resolve conflicts

File: login-workflow/src/screens/EulaScreen/EulaScreen.tsx [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+            showRefreshButton: eulaFetchError,
+            onRefresh: (): void => {
+                setEulaFetchError(false);
+                void loadAndCacheEula();
+            },
+        },
+    } = props;
+
+    const workflowCardHeaderProps = {
+        title: t('bluiRegistration:REGISTRATION.STEPS.LICENSE'),
```
Comment (manojleaton) [inline login-workflow/src/screens/EulaScreen/EulaScreen.tsx:164]: Please add default implementation for `onIconPress` function.  [reference](https://github.com/etn-ccis/blui-react-native-workflows/blob/b46c1b541cda3533e68d59b3d8209c33c06b042c/login-workflow/src/screens/VerifyCodeScreen/VerifyCodeScreen.tsx#L112)

Comment (ArshdeepSingh3-eaton) [reply login-workflow/src/screens/EulaScreen/EulaScreen.tsx:164]: what should be the operation for onIconPress in case of Eula Screen?

**Review by surajeaton — APPROVED** [review]
Looks good.

**Pull request merged**